www/main/settings/utils/theme-bg-reader.js

```diff
@@ -29,8 +29,8 @@ export const upsertCustomBackgrounds = (responseCallback = () => {}) => {
           .map((file) => ({
             name: file,
             path: `${userBackgroundsPath}/${file}`,
-            'background-image': `${userBackgroundsURL}/${file.replace(/(\s)/g, '\\ ')}`,
-            'background-image-path': `${userBackgroundsPath}/${file.replace(/(\s)/g, '\\ ')}`,
+            'background-image': `${userBackgroundsURL}/${file}`,
+            'background-image-path': `${userBackgroundsPath}/${file}`,
```

Comment:
Veerji, this regex was responsible for escaping white spaces in the images that are being uploaded. How are we handling this condition alternatively with this code?

Reply:
fs module handles this on its own. This regex fails in mac, and throws an error. I am still looking for edge cases and would update if needed.

Reply:
Fixed the edge case with recent commits.